modules/sdk-core/src/bitgo/utils/tss/eddsa/eddsa.ts

```diff
@@ -493,6 +493,8 @@ export class EddsaUtils extends baseTSSUtils<KeyShare> {
       txRequestId = txRequest.txRequestId;
     }
 
+    const { apiVersion } = txRequestResolved;
```

Comment:
apiVersion seems to be marked as an optional parameter in the type TxRequest. What do we do if it is not provided.

Reply:
`txRequestResolved` is actually a `txRequest` we get from wallet-platform or from the build result, so is not optional like the `buildParams` `apiVersion`